Patches/MeetingHudPatch.cs

```diff
@@ -736,8 +736,10 @@ public static bool Prefix(MeetingHud __instance, byte srcPlayerId, byte suspectP
         }
 
         // Coven Necronomicon Voting
-
-        if (suspectPlayerId == 253 && voter.IsPlayerCoven())
+        if (voter.IsPlayerCoven() && Main.AllAlivePlayerControls.Where(x => x.IsPlayerCoven()).Count() == 1) {
```

Comment:
`Where` and `Count` can be combined because `Count`, `Any`, ect. can perform the `Where` function
So it's better to use
```suggestion
        if (voter.IsPlayerCoven() && Main.AllAlivePlayerControls.Count(x => x.IsPlayerCoven()) == 1) {
```